akka-distributed-cluster-docs/src/main/paradox/guide/3-active-active.md

```diff
@@ -1,10 +1,102 @@
 # Part 3: Active-active
 
-For an Active-Active shopping cart, the same service will run in different data centers or cloud regions - each called a replica.
+Active-active means that the same shopping cart is in memory in multiple locations, replicas, and can accept updates
+in all of those locations, so that a cloud region outage does not block users from continuing to use the service.  
+
+@extref[Akka Replicated Event Sourcing](akka:typed/replicated-eventsourcing.html) stores persisted events in a local database,
+without any need for replication capabilities in the database itself, the events are then replicated using gRPC
+transport from @extref[Akka Projection gRPC](akka-projection:grpc-replicated-event-sourcing-transport.html).
```

Comment:
Should that be [Akka Projection gRPC] or [Akka Replicated Event Sourcing over gRPC] ?